jscomp/frontend/bs_builtin_ppx.ml

```diff
@@ -532,6 +545,42 @@ let rec structure_mapper ~await_context (self : mapper) (stru : Ast_structure.t)
             };
       }
       :: structure_mapper ~await_context self rest
+    | Pstr_value (_, vbs) ->
+      let item = self.structure_item self item in
+      let module_exprs =
+        vbs
+        |> List.filter_map (fun ({pvb_expr} : Parsetree.value_binding) ->
+               match pvb_expr.pexp_desc with
+               | Pexp_letmodule (_, ({pmod_attributes} as me), _)
+                 when Res_parsetree_viewer.hasAwaitAttribute pmod_attributes ->
+                 Some me
+               | _ -> None)
+      in
+      (* [ module __Belt_List__ = module type of Belt.List ] *)
+      let module_type_decls =
+        module_exprs
+        |> List.filter_map (fun ({pmod_desc} as me : Parsetree.module_expr) ->
+               match pmod_desc with
+               | Pmod_ident {txt; loc} -> (
+                 let safe_module_type_name = local_module_type_name txt in
+                 let has_local_module_name =
+                   Hashtbl.find_opt !await_context safe_module_type_name
+                 in
+
+                 match has_local_module_name with
+                 | Some _ -> None
+                 | None ->
+                   let open Ast_helper in
```

Comment:
nit: remove the open, just makes the code more indirect

Reply:
Not sure what the indirect means: https://github.com/rescript-lang/rescript-compiler/pull/6310/commits/1ed974fab4d2b41002ffba74aac91c12191447a7